### PR TITLE
Helm

### PR DESCRIPTION
**Pull request by adreghiciu**
* a new generic upload file operation that pushes helm charts that are randomly generated
* a new paths impl that is looping & downloads all charts based on helm index. The index is fetched every time all the charts are once downloaded and such also exercising the index pull

### CODE REVIEW

File: src/main/java/com/sonatype/nexus/perftest/helm/ChartGenerator.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      @Override
+      public boolean hasNext() {
+        return true;
+      }
+
+      @Override
+      public UploadEntity next() {
+        String name = "generator";
+        String version = "0.0." + ThreadLocalRandom.current().nextInt(1, numberOfCharts + 1);
+        ;
```
Comment (cstamas) [inline src/main/java/com/sonatype/nexus/perftest/helm/ChartGenerator.java]: ?

Comment (adreghiciu) [inline src/main/java/com/sonatype/nexus/perftest/helm/ChartGenerator.java]: The ; ?

Comment (adreghiciu) [inline src/main/java/com/sonatype/nexus/perftest/helm/ChartGenerator.java]: Removed

**Comment by cstamas** [comment]
+1

**Pull request merged**